### ISSUE TITLE
Log out button is not labeled for screenreader

### ISSUE THREAD

**Issue by heull001** [haiwen/seahub#5480]
the additional log out icon in topbar, configurable with SHOW_LOGOUT_ICON, has no link-text and so is not labeled for screenreader. So screenreader users don't no what it is for.

**Comment by Julia-1994** [comment]
Can you give us a screenshot of the additional log out icon in top bar ?
